test_cli.py

```diff
@@ -101,25 +101,22 @@ def get_uuid(line):
 
 
 def sanitize(string, max_chars=256):
-    try:
-        string.decode('utf-8')
-    except UnicodeDecodeError:
-        return '<binary>\n'
-
-    string = string.decode('utf-8').encode('ascii', errors='replace')  # Travis only prints ASCII
+    if type(string) is bytes:
+        string = "{}".format(string)
```

Comment:
For some reason this shows up as `<binary>` on the old travis CI runs, but on the new travis CI runs it ends up showing the entire (truncated) binary file. Do we know why?